### DISCUSSION TITLE
Language Detection Script (LUA)

### DISCUSSION THREAD

**Discussion by xenix4845** [YimMenu/YimMenu#2942] [Q&A]
I want to communicate with users of a specific language, but it is cumbersome to check with the information tab of the existing YIMMENU. Therefore, if a player using a specific language is detected by the lua script, I would like to inform the player's name and the language in use through the notification tab of YIMMENU gui. 

I'm not sure how to make it, so I'm looking for help.

**Comment by gamemonster1120** [comment]
hello.  We are co-developers developing a project together.  We are Korean.  We just want to find Korean players, but since the number of Korean players is small and it is difficult to find them, we just want to create a function to find Korean players.  Since it is inconvenient to search through the player list one by one to find Korean players, we would like to add a simple function that displays which players in the session are Korean through a message, so please advise once again.

**Reply by xenix4845** [reply]
I'm hoping for someone's help if it's possible.
